### PR TITLE
ci: Run feature_block in valgrind

### PR DESCRIPTION
**Pull request by maflcko**
Also revert commit 0a4912e46aecef6f04c6287508020ec60424ed24, because some tests take too long for this to be useful anymore.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Open-Close to re-run ci. See https://github.com/bitcoin/bitcoin/issues/15847#issuecomment-487591130

**Comment by practicalswift** [comment]
Concept ACK: we need to check for the use of uninitialized memory (UUM) in all tests

This one fixes `feature_block` which is good. What about the remaining two -- `feature_abortnode` and `rpc_bind`?

**Comment by MarcoFalke** [comment]
Added a commit to enable feature_abortnode as requested by @practicalswift . Can't enable rpc_bind because of this bug here: https://github.com/bitcoin/bitcoin/issues/17765#issuecomment-602068547

File: ci/test/00_setup_env_native_valgrind.sh [hunk 1]
```diff
@@ -10,7 +10,8 @@ export CONTAINER_NAME=ci_native_valgrind
 export PACKAGES="valgrind clang llvm python3-zmq libevent-dev bsdmainutils libboost-system-dev libboost-filesystem-dev libboost-test-dev libboost-thread-dev libdb5.3++-dev libminiupnpc-dev libzmq3-dev"
 export USE_VALGRIND=1
 export NO_DEPENDS=1
-export TEST_RUNNER_EXTRA="--exclude feature_abortnode,feature_block,rpc_bind"  # Excluded for now
+export TEST_RUNNER_EXTRA="--exclude rpc_bind"  # Excluded for now
+export RUN_UNIT_TESTS=true
```
Comment (practicalswift) [inline ci/test/00_setup_env_native_valgrind.sh]: Why disabling the unit tests? :)

**Comment by practicalswift** [comment]
@MarcoFalke Why is `RUN_FUNCTIONAL_TESTS=true` removed? :)

**Comment by MarcoFalke** [comment]
It is the default value, no need to set it to true twice.

```
$ git grep RUN_FUNCTIONAL_TESTS
ci/test/00_setup_env.sh:export RUN_FUNCTIONAL_TESTS=${RUN_FUNCTIONAL_TESTS:-true}

**Pull request merged**